bin/polymer.js

```diff
@@ -26,5 +26,6 @@ resolve('polymer-cli', {basedir: process.cwd()}, function(error, path) {
   let lib = path ? require(path) : require('..');
   let args = process.argv.slice(2);
   let cli = new lib.PolymerCli(args);
-  cli.run();
+  cli.run()
+  .then(null, () => process.exit(1));
```

Comment:
+1, using `then()` like this is fine, but we use `catch()` throughout the rest of the project. Lets stick to that here.

Also, can you please log a message before exit? Check out the `process.on('unhandledRejection', ...` handler in `polymer-cli.ts` for an example of what you'll need.